src/mumble/Settings.h

```diff
@@ -190,6 +190,7 @@ struct Settings {
 	};
 	enum WindowLayout { LayoutClassic, LayoutStacked, LayoutHybrid, LayoutCustom };
 	enum AlwaysOnTopBehaviour { OnTopNever, OnTopAlways, OnTopInMinimal, OnTopInNormal };
+	enum WhenConnectedQuitBehaviour { WhenConnectedQuitBehaviourAsk, WhenConnectedQuitBehaviourMinimize, WhenConnectedQuitBehaviourClose };
```

Comment:
We should turn this into an enum class and also move this to the global namespace instead of nesting in the Settings namespace. Maybe it would even be cleanest to put the definition in its own, dedicated header file.

Reply:
Is this a new way of implementation? I took the "AlwaysOnTopBehaviour" enum as a 'template' for the new setting. So I implemented everything like "AlwaysOnTopBehaviour".

Reply:
Yes - we just don't have to man-hours to refactor all of this legacy code, so all we can do is graually change the code, paying particular attention to new code.